sources/packages/api/src-sql/EducationProgramsOfferings/Create-education-programs-offerings.sql

```diff
@@ -0,0 +1,54 @@
+CREATE TABLE IF NOT EXISTS education_programs_offerings(
+  id SERIAL PRIMARY KEY,
+  offering_name VARCHAR(300) NOT NULL,
+  study_start_date Date NULL,
+  study_end_date Date NULL,
+  break_start_date Date NULL,
+  break_end_date Date NULL,
+  actual_tution_costs INT NULL,
+  program_related_costs INT NULL,
+  mandatory_fees INT NULL,
+  exceptional_expenses INT NULL, 
+  amount_requested INT NULL,
+  offering_delivered VARCHAR(50) NULL, 
+  lacks_program_dates BIT NOT NULL,
+  lacks_study_breaks BIT NOT NULL,
+  lacks_fixed_costs BIT NOT NULL,
+  tution_remittance_requested VARCHAR(50) NOT NULL,
```

Comment:
this checkbox right? so I guess it should be boolean field
![image](https://user-images.githubusercontent.com/77353155/121757434-80164300-caeb-11eb-9f3a-fc64390a54e1.png)

Reply:
We changed it to Yes or No Ann